src/code/z_kanfont.c

```diff
@@ -38,8 +38,8 @@ void Font_LoadOrderedFont(Font* font) {
     s32 fontBufIndex;
     u32 offset;
 
-    font->msgOffset = _message_0xFFFC_nes - (const char*)_nes_message_data_staticSegmentStart;
-    len = font->msgLength = _message_0xFFFD_nes - _message_0xFFFC_nes;
+    font->msgOffset = FONT_MESSAGE_START - (const char*)_nes_message_data_staticSegmentStart;
+    len = font->msgLength = FONT_MESSAGE_END - FONT_MESSAGE_START;
 
     DMA_REQUEST_SYNC(font->msgBuf, (uintptr_t)_nes_message_data_staticSegmentRomStart + font->msgOffset, len,
```

Comment:
tbh I'm not sure FONT_MESSAGE_START/FONT_MESSAGE_END is useful, I think it would probably be clearer if we ifdef this whole block since there are other changes too (msgBuf vs msgBufWide, extra division by 2, and jpn vs nes in _nes_message_data_staticSegmentRomStart and _nes_message_data_staticSegmentRomStart)

Reply:
I've always disliked how these symbols were exposed here before, I think using macros for them rather than the raw symbol is an improvement. I've changed the defines to OFFSET and LENGTH to subsume the segment start symbol at least, but yeah this will need some more work when matching this function.

Reply:
Personally I think the macros obscure what's going on, and it would be easier to read if it was more explicit. But yeah let's revisit later when matching